src/js/navigation.js

```diff
@@ -2,12 +2,54 @@ const navigationToggle = document.querySelector(".menu-btn");
 const navigation = document.querySelector("nav");
 const welcome = document.querySelector(".welcome");
 
-navigationToggle.addEventListener("click", ()  => {
+navigationToggle.addEventListener("click", () => {
   document.body.classList.toggle("nav-is-open");
+  welcome.classList.toggle("welcome-to-right");
   welcome.style.transition = "transform ease-out 250ms";
+  if (welcome.classList.contains("welcome-to-right")) {
+    const welcomeRight = document.querySelector(".welcome-to-right");
+    setTimeout(
```

Comment:
Whay do you need a timeot here?

Reply:
Če ni timeouta, potem event listener na welcomeRight istočasno, ko prvi event listener doda nav-is-open in welcome-to-right, ta dva razreda odstrani. Posledica: nav se sploh ne odpre.